ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/core/entities/classes/ConstructorInformation.java

```diff
@@ -0,0 +1,15 @@
+package edu.usb.argos.ASTProcessor.visitor.core.entities.classes;
+
+import edu.usb.argos.ASTProcessor.visitor.core.interfaces.nodes.Statement;
+import lombok.Builder;
+import lombok.Value;
+import java.util.List;
+
+@Value
+@Builder
+public class ConstructorInformation<S> {
+    private String name;
```

Comment:
remove the access modifier since the `@Value` already has that functionality

Reply:
Done